target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -202,6 +202,9 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "3:lan" "2:wan"
 		;;
+	compex,wpj558-16m)
+		ucidef_add_switch "switch0" "1:lan" "5:wan" "6@eth0"
```

Comment:
Please use the same formatting as everybody else.

Reply:
I do not understand the question? Test which ports are connected to the switch and then assign them so LAN and WAN is at the same ports as with stock firmware. PoE should not be relevant at all here?

Reply:
ar71xx was using opposite assignment: 5:lan 1:wan ...

Reply:
You only use one eth here although you set up two in DTS. This needs further work ...

Reply:
eth1 must be wired to port 0 because it is the last available port for RGMII on AR8327